scripts/asm-installer/install_asm

```diff
@@ -707,27 +713,108 @@ check_istio_not_deployed(){
 
 validate_istio_version() {
   info "Checking existing Istio version(s)..."
-  local OUTPUT
-  OUTPUT="$(retry 3 ./istio-${RELEASE}/bin/istioctl version -o json)"
-  local FOUND_VALID_VERSION
-  FOUND_VALID_VERSION="false"
-  for version in $(echo "${OUTPUT}" | jq -r '.meshVersion[].Info.version' -r); do
+  local VERSION_OUTPUT; VERSION_OUTPUT="$(retry 3 ./${ISTIOCTL_REL_PATH} version -o json)"
```

Comment:
I missed this, but we should check that VERSION_OUTPUT isn't blank because istioctl messed up somehow. Just a quick

```
if [[ -z "${VERSION_OUTPUT}" ]]; then
  fatal "Couldn't validate existing Istio versions."
fi
```